### ISSUE TITLE
[Clinic] On a followup visit, after vitals, RELEVANT ASPECTS does not have a + icon

### ISSUE THREAD

**Issue by kml27** [esaude/esaude-emr-poc#465]
Actual Results
--
Form for RELEVANT ASPECTS does not have a + icon
 

Expected results
--
forms for 
Under consultation tab:
OMS STADIUM
RELEVANT ASPECTS
ANAMNESIS AND PHYSICAL EXAMINATION 
DIAGNOSIS
Under the prescription tab:
PRESCRIPTION
are all available after vitals have been done for a new check-in on a followup visit

Steps to reproduce
--
After a followup check in, click clinical, click consultation, click vitals, enter vitals, next and review vitals, save vitals, notice no + next to relevant aspects form

Screenshot/Attachment (Optional)
--

**Comment by Guimino** [comment]
Hi @kml27  , I couldn't replicate this bug, Is it possible to add a screenshot for the last step?

**Comment by Guimino** [comment]
Hi @kml27 , I couldn't replicate this bug, Is it possible to add a screenshot for the last step?

**Comment by kml27** [comment]
I've tried to replicate this to provide a screenshot, following just the steps from my test scenario when I noticed it last time and it did not re-occur. When it first occurred, another test had recently been conducted by someone else, and added data to the patient. It may be steps from that scenario are required to recreate this bug. I will provide more information when I am able. (The test scenarios would have been conducted 18/4-19/4).
